cibuildwheel/resources/build-platforms.toml

```diff
@@ -28,10 +28,10 @@ python_configurations = [
 python_configurations = [
   { identifier = "cp36-macosx_x86_64", version = "3.6", url = "https://www.python.org/ftp/python/3.6.8/python-3.6.8-macosx10.9.pkg" },
   { identifier = "cp37-macosx_x86_64", version = "3.7", url = "https://www.python.org/ftp/python/3.7.9/python-3.7.9-macosx10.9.pkg" },
-  { identifier = "cp38-macosx_x86_64", version = "3.8", url = "https://www.python.org/ftp/python/3.8.9/python-3.8.9-macosx10.9.pkg" },
-  { identifier = "cp39-macosx_x86_64", version = "3.9", url = "https://www.python.org/ftp/python/3.9.4/python-3.9.4-macos11.pkg" },
-  { identifier = "cp39-macosx_arm64", version = "3.9", url = "https://www.python.org/ftp/python/3.9.4/python-3.9.4-macos11.pkg" },
-  { identifier = "cp39-macosx_universal2", version = "3.9", url = "https://www.python.org/ftp/python/3.9.4/python-3.9.4-macos11.pkg" },
+  { identifier = "cp38-macosx_x86_64", version = "3.8", url = "https://www.python.org/ftp/python/3.8.10/python-3.8.10-macos11.pkg" },
```

Comment:
Wow, looks like we have 3.8 universal finally! That would be great, as the built in AS Python is 3.8.

Reply:
Yep, it's official: https://pythoninsider.blogspot.com/2021/05/python-3810-395-and-3100b1-are-now.html?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+PythonInsider+%28Python+Insider%29

So a follow up would be to expand this to 3.8 and make the check in macOS.py check for 3.8+ instead of just string comparing 39. (Also needed for 3.10 soonish). Could make it an exclusion from `{"36", "37"}`, actually.

Reply:
Compared to the 3.9 installer, it's marked experimental on the [release page](https://www.python.org/downloads/release/python-3810/) and doesn't mention 10.9 as the minimal macOS on x86_64, only 11.0 BigSur. If I'm reading the logs from azure macOS correctly, the tests unfortunately seem to confirm 11.0 as the minimal macOS version.

Reply:
I see a pip resolver error on Azure, but I see something similar on the CircleCI mac tests- `Symbol not found: ____chkstk_darwin`

> Traceback (most recent call last):
>   File "/Users/distiller/project/cibuildwheel/resources/install_certifi.py", line 11, in <module>
>     import ssl
>   File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ssl.py", line 98, in <module>
>     import _ssl             # if we can't import it, let the error propagate
> ImportError: dlopen(/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/lib-dynload/_ssl.cpython-38-darwin.so, 2): Symbol not found: ____chkstk_darwin
>   Referenced from: /Library/Frameworks/Python.framework/Versions/3.8/lib/libcrypto.1.1.dylib (which was built for Mac OS X 11.0)
>   Expected in: /usr/lib/libSystem.B.dylib
>  in /Library/Frameworks/Python.framework/Versions/3.8/lib/libcrypto.1.1.dylib
> Error: Command ['sudo', '/Library/Frameworks/Python.framework/Versions/3.8/bin/python3', '/Users/distiller/project/cibuildwheel/resources/install_certifi.py'] failed with code 1. None

Looks like it was built without a `MACOSX_DEPLOYMENT_TARGET`. Oh well. I guess we can't use this version without macos-11 runners.